lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1649,6 +1667,15 @@ def _get_expected_ulimits(self, config_ulimits):
             ))
         return results
 
+    def _get_expected_sysctls(self, config_sysctls):
+        self.log('_get_expected_sysctls')
+        if config_sysctls is None:
+            return None
+        result = dict()
```

Comment:
<nod>  I use this instead of a dict comprehension:
``` python
return dict((key, str(value)) for key, value in config_systctls.items())
```

It's not more readable than the expanded for loop, though, so merging as is.